### PR TITLE
feat: add polish locales

### PR DESCRIPTION
**Pull request by veteran29**
## Related Issue or Design Document

- none

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/locales/pl.json [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1010005": "Zweryifkuj",
+  "identities.messages.1010006": "Kod autentykacyjny",
+  "identities.messages.1010007": "Zapasowe kody odzyskiwania",
+  "identities.messages.1010008": "Użyj klucza bezpieczeństwa",
+  "identities.messages.1010009": "Użyj Autentykatora",
+  "identities.messages.1010010": "Użyj zapasowych kodów odzyskiwania",
+  "identities.messages.1010011": "Kontynuuj za pomocą klucza bezpieczeństwa",
+  "identities.messages.1010012": "Przygotuj swoje urządzenie WebAuthn (np. klucz bezpieczeństwa, czytnik biometryczny, ...) a następnie kliknij kontynuuj.",
+  "identities.messages.1010013": "Kontynuuj",
+  "identities.messages.1010014": "Email zawierający kod został wysłany na wskazany adres email. Jeżeli nie otrzymano wiadomości to sprawdź pisownię adresu i spróbuj zalogować się ponownie.",
```
Comment (Demonsthere) [inline src/locales/pl.json:29]:
```suggestion
  "identities.messages.1010014": "Email zawierający kod został wysłany na wskazany adres email. Jeżeli nie otrzymasz wiadomości to sprawdź pisownię adresu i spróbuj zalogować się ponownie.",
```

File: src/locales/pl.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1010015": "Zaloguj się kodem",
+  "identities.messages.1040001": "Zarejestruj się",
+  "identities.messages.1040002": "Zarejestruj się za pomocą {provider}",
+  "identities.messages.1040003": "Kontynuuj",
+  "identities.messages.1040004": "Zarejestruj się kluczem bezpieczeństwa",
+  "identities.messages.1040005": "Email zawierający kod został wysłany na wskazany adres email. Jeżeli nie otrzymano wiadomości to sprawdź pisownię adresu i ponów rejestrację.",
```
Comment (Demonsthere) [inline src/locales/pl.json:35]:
```suggestion
  "identities.messages.1040005": "Email zawierający kod został wysłany na wskazany adres email. Jeżeli nie otrzymasz wiadomości to sprawdź pisownię adresu i ponów rejestrację.",
```

File: src/locales/pl.json [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1050010": "To są Twoje zapasowe kody odzyskiwania. Trzymaj je w bezpiecznym miejscu!",
+  "identities.messages.1050011": "Potwierdź zapasowe kody odzyskiwania",
+  "identities.messages.1050012": "Dodaj klucz bezpieczeństwa",
+  "identities.messages.1050013": "Nazwa klucza bezpieczeństwa",
+  "identities.messages.1050014": "<del>Wykorzystany</del>",
+  "identities.messages.1050015": "{secrets_list}",
+  "identities.messages.1050016": "Wyłącz tą metodę",
+  "identities.messages.1050017": "To jest sekret twojej aplikacji autentykator. Użyj go jeżeli nie możesz zeskanować kodu QR.",
+  "identities.messages.1050018": "Usuń klucz bezpieczeństwa \"{display_name}\"",
+  "identities.messages.1060001": "Pomyślnie odzyskałeś swoje konto. Proszę zmienić swoję hasło lub ustanowić alternatywną methodę logowania (np. logowanie zewnętrzne) w ciągu najbliższych {privileged_session_expires_at_unix_until_minutes} minut.",
```
Comment (Demonsthere) [inline src/locales/pl.json:55]:
```suggestion
  "identities.messages.1060001": "Pomyślnie odzyskałeś swoje konto. Proszę zmień swoję hasło lub ustanowić alternatywną methodę logowania (np. logowanie zewnętrzne) w ciągu najbliższych {privileged_session_expires_at_unix_until_minutes} minut.",
```

File: src/locales/pl.json [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1060002": "Email zawierający link odzyskiwania został wysłany na wskazany adres email. Jeżeli nie otrzymano wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```
Comment (Demonsthere) [inline src/locales/pl.json:56]:
```suggestion
  "identities.messages.1060002": "Email zawierający link odzyskiwania został wysłany na wskazany adres email. Jeżeli nie otrzymasz wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```

File: src/locales/pl.json [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1060003": "Email zawierający kod odzyskiwania został wysłany na wskazany adres email. Jeżeli nie otrzymano wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```
Comment (Demonsthere) [inline src/locales/pl.json:57]:
```suggestion
  "identities.messages.1060003": "Email zawierający kod odzyskiwania został wysłany na wskazany adres email. Jeżeli nie otrzymasz wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```

File: src/locales/pl.json [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1070005": "Wyślij",
+  "identities.messages.1070006": "Zweryfikuj kod",
+  "identities.messages.1070007": "Email",
+  "identities.messages.1070008": "Wyślij kod ponownie",
+  "identities.messages.1070009": "Kontynuuj",
+  "identities.messages.1070010": "Kod odzyskiwania",
+  "identities.messages.1070011": "Kod weryfikacyjny",
+  "identities.messages.1070012": "Kod rejestracji",
+  "identities.messages.1070013": "Kod logowania",
+  "identities.messages.1080001": "Email zawierający link weryfikacyjny został wysłany na wskazany adres email. Jeżeli nie otrzymano wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```
Comment (Demonsthere) [inline src/locales/pl.json:71]:
```suggestion
  "identities.messages.1080001": "Email zawierający link weryfikacyjny został wysłany na wskazany adres email. Jeżeli nie otrzymasz wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```

File: src/locales/pl.json [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.1080002": "Poprawnie zweryfikowano adres email.",
+  "identities.messages.1080003": "Email zawierający kod weryfikacyjny został wysłany na wskazany adres email. Jeżeli nie otrzymano wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```
Comment (Demonsthere) [inline src/locales/pl.json:73]:
```suggestion
  "identities.messages.1080003": "Email zawierający kod weryfikacyjny został wysłany na wskazany adres email. Jeżeli nie otrzymasz wiadomości to sprawdź pisownię adresu i upewnij się że używasz adresu użytego do rejestracji.",
```

File: src/locales/pl.json [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4000001": "{reason}",
+  "identities.messages.4000002": "Brakuje pola {property}.",
+  "identities.messages.4000003": "długość musi być >= {min_length}, otrzymano {actual_length}",
+  "identities.messages.4000004": "nie pasuje do wzoru \"{pattern}\"",
+  "identities.messages.4000005": "Hasło nie może być użyte ponieważ {reason}.",
+  "identities.messages.4000006": "Podane poświadczenia są nieprawidłowe, sprawdź czy nie ma literówek w twoim haśle, nazwie użytkownika, adresie email lub numerze telefonu.",
+  "identities.messages.4000007": "Istnieje już konto z takim samym identyfikatorem (email, telefon, nazwa użytkownika, ...).",
```
Comment (Demonsthere) [inline src/locales/pl.json:80]:
```suggestion
  "identities.messages.4000007": "Istnieje już konto o takim samym identyfikatorze (email, telefon, nazwa użytkownika, ...).",
```

File: src/locales/pl.json [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4000008": "Podany kod autentykacyjny jest nieprawidłowy, spróbuj ponownie.",
+  "identities.messages.4000009": "Nie odnaleziono żadnych identyfikatorów logowania. Czy zapomniałeś je ustawić? Może też być to spowodowane nieprawidłową konfiguracją serwera.",
+  "identities.messages.4000010": "Konto nie jest jeszcze aktywne. Czy zapomniałeś zweryfikować swój adres email?",
```
Comment (Demonsthere) [inline src/locales/pl.json:83]:
```suggestion
  "identities.messages.4000010": "Konto nie jest jeszcze aktywne. Czy zapomniałeś/łaś zweryfikować swój adres email?",
```

File: src/locales/pl.json [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4000018": "musi być >= {minimum} otrzymano {actual}",
+  "identities.messages.4000019": "musi być > {minimum} otrzymano {actual}",
+  "identities.messages.4000020": "musi być <= {maximum} otrzymano {actual}",
+  "identities.messages.4000021": "musi być < {maximum} otrzymano {actual}",
+  "identities.messages.4000022": "{actual} nie jest wielokrotnością {base}",
+  "identities.messages.4000023": "maksymalnie {max_items} wpisów, otrzymano {actual_items} wpisów",
+  "identities.messages.4000024": "minimum {min_items} wpisów, otrzymano {actual_items} wpisów",
+  "identities.messages.4000025": "wpisów w indeksie {index_a} i {index_b} są takie same",
+  "identities.messages.4000026": "spodziewano {allowed_types_list}, otrzymano {actual_type}",
+  "identities.messages.4000027": "Istnieje już konto z takim samym identyfikatorem (email, telefon, nazwa użytkownika, ...). Zaloguj się do swojego konta i połącz swój profil zewnętrzny na stronie ustawień.",
```
Comment (Demonsthere) [inline src/locales/pl.json:100]:
```suggestion
  "identities.messages.4000027": "Istnieje już konto o takim samym identyfikatorze (email, telefon, nazwa użytkownika, ...). Zaloguj się do swojego konta i połącz swój profil zewnętrzny na stronie ustawień.",
```

File: src/locales/pl.json [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4000028": "Próbowano się zalogować za pomocą {credential_identifier_hint} który jest już używany przez inne konto. Możesz się zalogować za pomocą {available_credential_types_list}. Możesz się zalogować za pomocą następujących zewnętrznch dostawców logowania: {Available_oidc_providers_list}.",
+  "identities.messages.4000029": "musi być równe stałej {expected}",
+  "identities.messages.4000030": "const nie powiodła się",
+  "identities.messages.4000031": "Hasło nie może być użytę ponieważ jest zbyt podobne do identyfikatora.",
+  "identities.messages.4000032": "Hasło musi być długie na co najmniej {min_length} znaków, wprowadzono {actual_length} znaków.",
+  "identities.messages.4000033": "Hasło musi być nie dłuższe niż {max_length} znaków, wprowadzono {actual_length} znaków.",
+  "identities.messages.4000034": "Hasło wyciekło w wyniku naruszenia danych i nie może być użyte.",
+  "identities.messages.4000035": "To konto nie istnieje lub nie pozwala na logowanie kodem.",
+  "identities.messages.4000036": "Podane cechy nie odpowiadają cechom wcześniej powiązanym z tym przepływem.",
```
Comment (Demonsthere) [inline src/locales/pl.json:109]:
```suggestion
  "identities.messages.4000036": "Podane atrybuty nie odpowiadają atrybutom wcześniej powiązanym z tym procesem.",
```

File: src/locales/pl.json [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010001": "Przepływ logowania wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```
Comment (Demonsthere) [inline src/locales/pl.json:110]:
```suggestion
  "identities.messages.4010001": "Proces logowania wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```

File: src/locales/pl.json [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010002": "Nie odnaleziono strategii do logowania. Czy formularz został poprawnie wypełniony??",
```
Comment (Demonsthere) [inline src/locales/pl.json:111]:
```suggestion
  "identities.messages.4010002": "Nie odnaleziono strategii do logowania. Czy formularz został poprawnie wypełniony?",
```

File: src/locales/pl.json [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010003": "Nie odnaleziono strategii do rejestracji. Czy formularz został poprawnie wypełniony??",
```
Comment (Demonsthere) [inline src/locales/pl.json:112]:
```suggestion
  "identities.messages.4010003": "Nie odnaleziono strategii do rejestracji. Czy formularz został poprawnie wypełniony?",
```

File: src/locales/pl.json [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010004": "Nie odnaleziono strategii do aktualizacji ustawień. Czy formularz został poprawnie wypełniony??",
```
Comment (Demonsthere) [inline src/locales/pl.json:113]:
```suggestion
  "identities.messages.4010004": "Nie odnaleziono strategii do aktualizacji ustawień. Czy formularz został poprawnie wypełniony?",
```

File: src/locales/pl.json [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010005": "Nie odnaleziono strategii do odzyskiwania konta. Czy formularz został poprawnie wypełniony??",
```
Comment (Demonsthere) [inline src/locales/pl.json:114]:
```suggestion
  "identities.messages.4010005": "Nie odnaleziono strategii do odzyskiwania konta. Czy formularz został poprawnie wypełniony?",
```

File: src/locales/pl.json [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010006": "Nie odnaleziono strategii do weryfikacj konta. Czy formularz został poprawnie wypełniony??",
```
Comment (Demonsthere) [inline src/locales/pl.json:115]:
```suggestion
  "identities.messages.4010006": "Nie odnaleziono strategii do weryfikacj konta. Czy formularz został poprawnie wypełniony?",
```

File: src/locales/pl.json [hunk 18]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4010007": "Żądanie zostało poprawnie zakończone i nie może być ponowione.",
+  "identities.messages.4010008": "Kod logowania jest niepoprawny lub został wykorzystany. Spróbuj ponownie.",
+  "identities.messages.4040001": "Przepływ rejestracji wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```
Comment (Demonsthere) [inline src/locales/pl.json:118]:
```suggestion
  "identities.messages.4040001": "Proces rejestracji wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```

File: src/locales/pl.json [hunk 19]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4040002": "Żądanie zostało poprawnie zakończone i nie może być ponowione.",
+  "identities.messages.4040003": "Kod rejestracji jest niepoprawny lub został wykorzystany. Spróbuj ponownie.",
+  "identities.messages.4050001": "Przepływ ustawień wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```
Comment (Demonsthere) [inline src/locales/pl.json:121]:
```suggestion
  "identities.messages.4050001": "Proces ustawień wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```

File: src/locales/pl.json [hunk 20]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4060001": "Żądanie zostało poprawnie zakończone i nie może być ponowione.",
+  "identities.messages.4060002": "Przepły odzyskiwania jest niepoprawny i musi zostać ponowiony.",
```
Comment (Demonsthere) [inline src/locales/pl.json:123]:
```suggestion
  "identities.messages.4060002": "Proces odzyskiwania jest niepoprawny i musi zostać ponowiony.",
```

File: src/locales/pl.json [hunk 21]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4060004": "Token odzyskiwania jest niepoprawny lub został wykorzystany. Proszę ponowić przepływ.",
```
Comment (Demonsthere) [inline src/locales/pl.json:124]:
```suggestion
  "identities.messages.4060004": "Token odzyskiwania jest niepoprawny lub został wykorzystany. Proszę ponowić proces.",
```

File: src/locales/pl.json [hunk 22]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4060005": "Przepływ odzyskiwania wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```
Comment (Demonsthere) [inline src/locales/pl.json:125]:
```suggestion
  "identities.messages.4060005": "Proces odzyskiwania wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```

File: src/locales/pl.json [hunk 23]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4060006": "Kod odzyskiwania jest nieprawidłowy lub został wykorzystany. Spróbuj ponownie.",
+  "identities.messages.4070001": "Token weryfikacyjny jest nieprawidłowy lub został wykorzystany. Proszę ponowić przepływ.",
```
Comment (Demonsthere) [inline src/locales/pl.json:127]:
```suggestion
  "identities.messages.4070001": "Token weryfikacyjny jest nieprawidłowy lub został wykorzystany. Proszę ponowić proces.",
```

File: src/locales/pl.json [hunk 24]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4070002": "Żądanie zostało poprawnie zakończone i nie może być ponowione.",
+  "identities.messages.4070003": "Przepływ weryfikacyjny jest nieprawidłowy i musi zostać ponowiony.",
```
Comment (Demonsthere) [inline src/locales/pl.json:129]:
```suggestion
  "identities.messages.4070003": "Proces weryfikacyjny jest nieprawidłowy i musi zostać ponowiony.",
```

File: src/locales/pl.json [hunk 25]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4070005": "Przepływ weryfikacyjny wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```
Comment (Demonsthere) [inline src/locales/pl.json:130]:
```suggestion
  "identities.messages.4070005": "Proces weryfikacyjny wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
```

File: src/locales/pl.json [hunk 26]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4070006": "Kod weryfikacyjny jest nieprawidłowy lub został wykorzystany. Spróbuj ponownie.",
+  "identities.messages.5000001": "{reason}",
+  "login.forgot-password": "Zapomniałeś hasła?",
```
Comment (Demonsthere) [inline src/locales/pl.json:133]:
```suggestion
  "login.forgot-password": "Zapomniałeś\łaś hasła?",
```

File: src/locales/pl.json [hunk 27]
```diff
@@ -0,0 +1,172 @@
...
+  "registration.subtitle-oauth2": "Do autentykacji {clientName}",
+  "registration.title": "Zarejestruj konto",
+  "settings.navigation-backup-codes": "Kody zapasowe 2FA",
+  "settings.navigation-logout": "Wyloguj",
+  "settings.navigation-oidc": "Zewnętrzne logowanie",
+  "settings.navigation-password": "Hasło",
+  "settings.navigation-profile": "Profil",
+  "settings.navigation-totp": "Aplikacja Autentykator",
+  "settings.navigation-webauthn": "Tokeny Sprzętowe",
+  "settings.subtitle-instructions": "Tutaj możesz zarządzać ustawieniami związnymi z twoim kontem. Miej na uwadze że niektórę akcje mogą wymagać pownownej autentykacji.",
```
Comment (Demonsthere) [inline src/locales/pl.json:160]:
```suggestion
  "settings.subtitle-instructions": "Tutaj możesz zarządzać ustawieniami związanymi z Twoim kontem. Miej na uwadze że niektóre akcje mogą wymagać ponownej uwierzytelnienia.",
```

**Review by Demonsthere — COMMENTED** [review]
Adding some suggestions :)

**Comment by veteran29** [comment]
Applied, thanks. Should've done them after bit more of sleep. 🥴

File: src/locales/pl.json [hunk 28]
```diff
@@ -0,0 +1,172 @@
...
+  "identities.messages.4060004": "Token odzyskiwania jest niepoprawny lub został wykorzystany. Proszę ponowić proces.",
+  "identities.messages.4060005": "Proces odzyskiwania wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
+  "identities.messages.4060006": "Kod odzyskiwania jest nieprawidłowy lub został wykorzystany. Spróbuj ponownie.",
+  "identities.messages.4070001": "Token weryfikacyjny jest nieprawidłowy lub został wykorzystany. Proszę ponowić proces.",
+  "identities.messages.4070002": "Żądanie zostało poprawnie zakończone i nie może być ponowione.",
+  "identities.messages.4070003": "Proces weryfikacyjny jest nieprawidłowy i musi zostać ponowiony.",
+  "identities.messages.4070005": "Proces weryfikacyjny wygasł {expired_at_unix_since_minutes} minut temu, spróbuj ponownie.",
+  "identities.messages.4070006": "Kod weryfikacyjny jest nieprawidłowy lub został wykorzystany. Spróbuj ponownie.",
+  "identities.messages.5000001": "{reason}",
+  "login.forgot-password": "Zapomniałeś\łaś hasła?",
```
Comment (Demonsthere) [inline src/locales/pl.json:133]: It seems the ci doesn't like this format 😞 
```suggestion
  "login.forgot-password": "Zapomniałeś/łaś hasła?",
```

Comment (veteran29) [reply src/locales/pl.json:133]: backslash is special character in JSON, should be escaped as `\\`

Comment (Demonsthere) [reply src/locales/pl.json]: True, but / is not a special character, and imho looks more natural than \ 😉

**Review by Demonsthere — APPROVED** [review]
LGTM :)

**Review by Benehiko — APPROVED** [review]

**Pull request merged**